src/evaluator/index.js

```diff
@@ -58,10 +58,10 @@ class TypedValue {
  * @property {radspec/Bindings} bindings
  */
 class Evaluator {
-  constructor (ast, bindings, { ethNode, to } = {}) {
+  constructor (ast, bindings, { ethNode, to, eth } = {}) {
     this.ast = ast
     this.bindings = bindings
-    this.eth = new Eth(ethNode || 'https://mainnet.infura.io')
+    this.eth = eth || new Eth(ethNode || 'https://mainnet.infura.io')
```

Comment:
Hmm, are we ever passing `eth` around? It should just be `ethNode`.

Reply:
Helpers have the `eth` object so they can do network requests, but they don't have the `ethNode`. I guess there is a method to get it, but this seemed cleaner to me.